Tools/LogAnalyzer/DataflashLog.py

```diff
@@ -501,7 +501,7 @@ def process(self, lineNumber, e):
             else:
                 self.messages[lineNumber] = e.Message
         elif e.NAME == "MODE":
-            if self.vehicleType == "ArduCopter":
+            if self.vehicleType in ["ArduCopter"]:
```

Comment:
That's indeed a remainder of trying to move in APM:Copter here (which did not work - see above). This particular change could be reverted therefore.